### PR TITLE
Add Gnosis Safe web UI

### PR DESCRIPTION
**Pull request by Uxio0**
Add Gnosis Safe Web UI. It still needs to be improved, as we are using the development server, but it's a first step

### CODE REVIEW

File: container_env_files/ui.env [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+REACT_APP_ETHGASSTATION_API_KEY=
+
+# Versions
+REACT_APP_LATEST_SAFE_VERSION=
+
+# Leave it untouched, version will set using dotenv-expand
+REACT_APP_APP_VERSION=$npm_package_version
+
+# Contracts Addresses
+REACT_APP_SPENDING_LIMIT_MODULE_ADDRESS=0x9e9Bf12b5a66c0f0A7435835e0365477E121B110
```
Comment (rmeissner) [inline container_env_files/ui.env:28]: Should this be set by default :thinking:

File: container_env_files/cgw.env [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # More detailed description of the available values can be found:
-# https://github.com/gnosis/safe-client-gateway/blob/main/.env.sample
+# https://github.com/gnosis/safe-client-gateway/blob/main/.env.example
```
Comment (rmeissner) [inline container_env_files/cgw.env:2]: the original was correct

File: README.md [hunk 3]
```diff
@@ -42,14 +42,14 @@ docker exec -it safe-infrastructure_db_1 psql -U postgres
 
 ## Step 1: setup your `.env`
 
-We recommend using what is available in the `.env.sample` file as it contains versions of our services guaranteed to be compatible with each other: 
+We recommend using what is available in the `.env.example` file as it contains versions of our services guaranteed to be compatible with each other:
```
Comment (rmeissner) [inline README.md:45]: The file name before was correct

File: README.md [hunk 4]
```diff
@@ -42,14 +42,14 @@ docker exec -it safe-infrastructure_db_1 psql -U postgres
...
 
 ```bash
-cp .env.sample .env
+cp .env.example .env
```
Comment (rmeissner) [inline README.md:48]: Same here

File: container_env_files/cgw.env (hunk 2 above)
Comment (Uxio0) [reply container_env_files/cgw.env:2]: Ok, I did it to align with `safe-react` repo

File: container_env_files/ui.env (hunk 1 above)
Comment (Uxio0) [reply container_env_files/ui.env:28]: I copied it from https://github.com/gnosis/safe-react/blob/dev/.env.example , I didn't want to break anything (cc @gnosis/safe-web)

Comment (iamacook) [reply container_env_files/ui.env:28]: We can definitely remove this from `.env.example`.

The spending limit module address we use (when not in Secrets) is [here](https://github.com/gnosis/safe-react/blob/dev/src/utils/constants.ts#L24).

File: README.md [hunk 5]
```diff
@@ -121,3 +121,7 @@ Then use `manage.py` and the custom command for adding a webhook like so:
 ```bash
 python manage.py add_webhook --url=http://nginx:8000/cgw/v1/hook/update/some_random_token
 ```
+
+4. Access the UI
+You can now access to the Gnosis Safe Web UI at http://localhost:8080. Port can be configured on `.env`
```
Comment (iamacook) [inline README.md:126]: "You can then access..."

**Review by iamacook — APPROVED** [review]
There's a sentence I'd adjust, but it's still correct.

File: README.md (hunk 5 above)
Comment (Uxio0) [reply README.md]: Makes sense, thanks @iamacook

Comment (Uxio0) [reply README.md]: Done

**Pull request merged**